microsoft-edge/web-platform/release-notes/135.md

```diff
@@ -0,0 +1,391 @@
+
+You can "subscribe" to this object to receive events as they come in, and call any of its operators/combinators to declaratively describe the flow of transformations through which events go. This is in contrast with the imperative version, which often requires complicated nesting with things like `addEventListener()`. For more on this, see the examples in the explainer.
+
+See the [W3C Observable specification](https://wicg.github.io/observable/).
+
+
+<!-- ---------- -->
+###### Service worker client URL now ignores `history.pushState` changes
+
+The `Client.url` property of a service worker now ignores document URL changes made by the `history.pushState()` method and other similar history APIs. The `Client.url` property is intended to be the creation URL of the HTML document which ignores such changes.
+
+
+<!-- ---------- -->
+###### Timestamps for RTC Encoded Frames
+
+Expose the following timestamps that are present in WebRTC encoded frames transmitted via `RTCPeerConnection`:
+
+* Capture timestamp: the timestamp when a frame was originally captured.
+* Receive timestamp: the timestamp when a frame was received.
+
+This change allows video conferencing applications which use WebRTC to implement latency measurements to better understand performance.
+
+
+<!-- ---------- -->
+###### Use the `double` type for ProgressEvent's `loaded` and `total` attributes
+
+The `loaded` and `total` attributes of a ProgressEvent indicate the current progress. The type of these attributes is now `double`, which gives developers more control over the value. For example, developers can now create a ProgressEvent with the value of `total` set to `1` and the the value of `loaded` increasing from `0` to `1` gradually. This aligns the event with the default behavior of the `<progress>` HTML element if the `max` attribute is omitted.
+
+
+<!-- ---------- -->
+###### fetchLater API
+
+`fetchLater()` is a JavaScript API to send a deferred fetch request. Once sent, a deferred request is queued by the browser in a _pending_ state, and will be invoked by the earliest of the following conditions:
+
+* The document is destroyed.
+* After a user-specified time.
+* When the browser decides it's time to send the request.
+
+The API returns a `FetchLaterResult` that contains the `activated` boolean field, which may be updated to tell whether the deferred request has been sent or not. On successful sending, the response is ignored by browser, including its body and headers.
+
+
+<!-- ---------- -->
+###### highlightsFromPoint API
+
+The `highlightsFromPoint()` API enables developers to interact with CSS Custom Highlights by detecting which highlights exist at a specific point within a document. Getting highlights from a point can be used by web apps to manage user interactions with custom highlights, such as responding to user clicks or hover events on highlighted regions to trigger custom tooltips or contextual menus.
```

Comment:
```suggestion
The `highlightsFromPoint()` API enables your webpages to interact with CSS Custom Highlights, by detecting which highlights exist at a specific point within a document.

Getting highlights from a point can be used by a web app to manage user interactions with custom highlights, such as responding to user clicks or hover events on a highlighted region, to trigger a custom tooltip or contextual menu.
```
good example of increasing the clarity by rewording from plural to singular
@captainbrosset